frontend/components/auction/debt/DebtAuction.stories.js

```diff
@@ -0,0 +1,81 @@
+import { storiesOf } from '@storybook/vue';
+import { action } from '@storybook/addon-actions';
+import DebtAuction from './DebtAuction';
+import { generateFakeSurplusAuctionTransaction } from '~/helpers/generateFakeSurplusAuction';
```

Comment:
This is a note for the future. Your currently use the `SurplusAuction` types. In #398 @KirillDogadin-std and I are finalising the types for the `DeptAuctions`, which are simular to the `SurplusAuction`, however there are slight differences. 

Once the PR is merged, you will need to update all the types.

Reply:
Debt Auction Types have now been merged!